### ISSUE TITLE
Bottom navigation bar overlapping apps sometimes

### ISSUE THREAD

**Issue by gefloh** [GrapheneOS/os-issue-tracker#4179]
Hi,

I am not using the gesture mode for navigating but the three buttons in the bar at the bottom end of the screen. In some applications I have the problem that the app is not respecting this bar, means the app takes the full screen size so that the navigation bar overlaps the lower part of the app which makes the app hard to use or not usable at all. In the example below the dark grey bar contains the OS navigation (the buttons themselves are not visible in the screenshots unfortunately) and it overlaps the yellow app buttons, which are not clickable in this case.

Not sure if this is a GrapheneOS issue or an app issue though, although I never had this issue with other Androids.

Pixel 8 Pro, latest stable GrapheneOS version.

![Screenshot_20241022-124632](https://github.com/user-attachments/assets/7da20715-b866-42bd-890f-39c959729991)

**Comment by gefloh** [comment]
> These apps updated their target API level to Android 15 without testing on an Android 15 device. Android 15 enabled edge-to-edge by default. It's unfortunate that app developers mark their app as targeting a new OS release without ever having tested it on a device. You have to report these app bugs on Android 15 to app developers. This would not occur if the app had left their app targeting Android 14 until they had time to test 15 properly.

Yes, understandable although this issue was also present in Android 14 for me unfortunately.

**Comment by gefloh** [comment]
Thanks!

**Comment by pcolaianni** [comment]
> These apps updated their target API level to Android 15 without testing on an Android 15 device. Android 15 enabled edge-to-edge by default. It's unfortunate that app developers mark their app as targeting a new OS release without ever having tested it on a device. You have to report these app bugs on Android 15 to app developers. This would not occur if the app had left their app targeting Android 14 until they had time to test 15 properly.

The same happens in the Settings app: the keyboard comes up, and the bottom is covered by the navigation keys. Is the Settings app compatible with Android 15?

All the apps I use seem to have this issue.

In Syncthing it is the opposite: the title of the app is covered by the status bar. Can't tap anything there.
